crates/exex/src/model.rs

```diff
@@ -360,6 +418,22 @@ mod tests {
             // Assert roundtrip conversion is equal to original value
             prop_assert_eq!(roundtrip_address, address);
         }
+
+        #[test]
+        fn test_bloom_to_keth_maybe_relocatable_roundtrip(bloom in any::<Bloom>()) {
+            // Convert Bloom to KethMaybeRelocatables
+            let keth_maybe_relocatables = KethMaybeRelocatables::from(bloom);
+
+            // Convert back to Bloom
+            let roundtrip_bloom = KethMaybeRelocatables::to_bloom(&keth_maybe_relocatables);
+
+            // Assert roundtrip conversion is equal to original value
+            prop_assert_eq!(roundtrip_bloom, bloom);
+            prop_assert_eq!(
+                bloom.len(),
+                keth_maybe_relocatables.0[0].0.get_int().unwrap().to_string().parse::<usize>().unwrap()
+            );
+        }
```

Comment:
@ClementWalter This test is quite heavy (few seconds to run) because the fuzzing on this 256 bits elements is quite huge to generate. To cover a little more I put below an additional test with `Bloom::random()` so we could "in theory" remove the proptest but I hesitate because the fuzzing is quite important on this tricky conversion, what do you think?

Reply:
So just confirming that a `Bloom` is 256 bytes long, ie 32bytes * 8 words
So in cairo this will be a Uint256*, and in terms of segment actually 16 felts

Reply:
in terms of testing, can't we just ask for few examples and not a lot ?

Reply:
@ClementWalter For testing I've just removed the proptest for this in favor of a loop over 100 random in a classical test which is way faster and similar in terms of security

Reply:
initial bloom is 16 times bytes16 (not sure why this format and not just `[u8]`). I don't know if the best representation for keth is `[u8]` (ie `felt*` with one byte per felt) or `[u128]` (ie `Uint256*` with `low, high, low,high, etc.)

let me check where it's used in the EVM (not used in kakarot currently)

Reply:
ok looking at https://github.com/ethereum/execution-specs/blob/9b95554a88d2a8485f8180254d0f6a493a593fda/src/ethereum/cancun/fork.py#L689

it seems like a list of u128 is fine. note sure to understand you reversed order manipulation though, let's say we have  `bloom: felt*`, we'll have

```
bloom[0] = int.from_bytes(bytes([149, 59, 225, 248, 7, 245, 195, 147, 171, 156, 97, 130, 30, 73, 17, 104]))
...
bloom[15] = int.from_bytes(bytes([155, 64, 120, 34, 76, 21, 10, 203, 23, 114, 21, 101, 59, 7, 111, 68]))
```

Reply:
Done